service/__init__.py

```diff
@@ -6,31 +6,39 @@
 """
 import sys
 from flask import Flask
+from flask_restx import Api
 from service import config
 from service.common import log_handlers
 
 # Create Flask application
 app = Flask(__name__)
 app.config.from_object(config)
-
+app.url_map.strict_slashes = False
+api = Api(
+    app,
+    version="1.0.0",
+    title="Inventory Demo REST API Service",
+    description="This is a sample Inventory store server.",
+    default="inventory",
+    default_label="Inventory warehouse operations",
+    doc="/apidocs",
+    prefix="/api",
+)
 # Dependencies require we import the routes AFTER the Flask app is created
 # pylint: disable=wrong-import-position, wrong-import-order, cyclic-import
 from service import routes, models  # noqa: E402, E261
-# pylint: disable=wrong-import-position
-from service.common import error_handlers, cli_commands  # noqa: F401, E402
 
+# pylint: disable=wrong-import-position
 # Set up logging for production
 log_handlers.init_logging(app, "gunicorn.error")
-
 app.logger.info(70 * "*")
 app.logger.info("  S E R V I C E   R U N N I N G  ".center(70, "*"))
 app.logger.info(70 * "*")
 
 try:
-    models.init_db(app)  # make our SQLAlchemy tables
+    routes.init_db(app)  # make our SQLAlchemy tables
```

Comment:
I may be missing something here. Can you tell why did you change `models` to `routes` while calling the `init_db` function?

Reply:
I copied the code from the restplus lab, and that had routes.init_db(app)

Reply:
It doesn't make a difference though